### PR TITLE
#27 - Function to sample a concrete matrix from an interval matrix

### LINKED ISSUE #27: Function to sample a concrete matrix from an interval matrix

### PR DESCRIPTION
**Pull request by schillic**
Closes #27.

### CODE REVIEW

File: src/matrix.jl [hunk 1]
```diff
@@ -131,3 +132,30 @@ function split(A::IntervalMatrix{T}) where {T}
...
+
+A concrete matrix.
+"""
+function rand(A::IntervalMatrix{T}; rng::AbstractRNG=GLOBAL_RNG) where {T}
+    m, n = size(A)
+    B = Matrix{T}(undef, m, n)
+    @inbounds for j in 1:n
+        for i in 1:m
+            itv = A[i, j]
+            B[i, j] = rand(Uniform(itv.left, itv.right))
```
Comment (mforets) [inline src/matrix.jl]: We can as well use `Base.random` here and not require `Distributions`, by letting `B[i, j] = (itv.right - itv.left)  * rand(rng) + itv.left`.

Comment (schillic) [reply src/matrix.jl]: Okay, I changed it.

File: test/runtests.jl [hunk 2]
```diff
@@ -49,4 +49,10 @@ end
     a2 = [0. -3.; 4. 0.]
     @test a1 ∈ m
     @test a2 ∉ m
+
+@testset "Interval matrix rand" begin
+    m = IntervalMatrix([-1.1..0.9 -4.1.. -3.9; 3.9..4.1 -1.1..0.9])
+    for i in 1:3
+        rand(m)
```
Comment (mforets) [inline test/runtests.jl]: we can now add a check:

```suggestion
        @test rand(m) ∈ m
```
(after rebasing with master)

File: src/matrix.jl [hunk 3]
```diff
@@ -146,4 +147,30 @@ function ∈(M::AbstractMatrix, A::AbstractIntervalMatrix)
         end
     end
     return true
+
```
Comment (mforets) [inline src/matrix.jl]: there is an `end` that got lost somewhere, i think in line 150

**Pull request merged**